che-incubator/chectl
# fix: Remove the process to check OIDC provider on server:deploy

### What does this PR do?

- Remove `--skip-oidc-provider-check` flag and its logic.

### Screenshot/screencast of this PR


### What issues does this PR fix or reference?


https://github.com/eclipse-che/che/issues/23047

### How to test this PR?


```bash
%./bin/run server:deploy --skip-oidc-provider-check
Error: Nonexistent flag: --skip-oidc-provider-check
```

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
